ClearKeep/Scenes/Profile/Views/UserProfileContentView.swift

```diff
@@ -0,0 +1,476 @@
+							}
+						}
+						
+						NavigationLink(destination: ChangePasswordView(),
+									   isActive: $isChangePassword) {
+							Button(action: changePassword) {
+								HStack {
+									Text("UserProfile.Password.Change".localized)
+										.font(AppTheme.shared.fontSet.font(style: .body3))
+										.foregroundColor(AppTheme.shared.colorSet.primaryDefault)
+									Spacer()
+									AppTheme.shared.imageSet.arrowRightIcon
+										.foregroundColor(AppTheme.shared.colorSet.primaryDefault)
+								}
+							}
+						}
+					}
+					
+					VStack(alignment: .leading, spacing: Constants.spacerTop) {
+						HStack(alignment: .top) {
+							VStack(alignment: .leading, spacing: Constants.spacerTop) {
+								Text("UserProfile.Authen.2FA".localized)
+									.font(AppTheme.shared.fontSet.font(style: .body2))
+									.foregroundColor(foregroundColorSetting)
+								Text("UserProfile.2FA.Title".localized)
+									.font(AppTheme.shared.fontSet.font(style: .input3))
+									.foregroundColor(foregroundColor)
+							}
+							Spacer()
+							
+							NavigationLink(destination: TwoFactorView(twoFactorType: .setting),
+										   isActive: $isCurrentPass) {
+								Button(action: change2FAStatus) {
+									Text(statusTwoFA)
+										.font(AppTheme.shared.fontSet.font(style: .body3))
+										.foregroundColor(AppTheme.shared.colorSet.primaryDefault)
+								}
+							}
+						}
+					}
+					Spacer()
+				}
+				.padding([.horizontal, .top], Constants.spacer)
+			}
+			.edgesIgnoringSafeArea(.all)
+			.applyNavigationGradientPlainStyle(title: "UserProfile.Setting".localized, leftBarItems: { buttonLeft }, rightBarItems: { rightButton })
+			.confirmationDialog("", isPresented: $showingImageOptions, titleVisibility: .hidden) {
+				Button("Chat.TakePhoto".localized) {
+					showingCameraPicker = true
+				}
+				Button("Chat.Albums".localized, role: .destructive) {
+					isImagePickerPresented = true
+				}
+				Button("Chat.Cancel".localized, role: .cancel) {
+				}
+			}
+			.fullScreenCover(isPresented: $isShowCountryCode, content: {
+				CountryCode(selectedNum: $countryCode)
+			})
+			.fullScreenCover(isPresented: $showingCameraPicker, content: {
+				CameraImagePicker(sourceType: .camera) { addImage in
+					selectedImages.removeAll()
+					self.selectedImages.append(addImage)
+					updateAvata()
+				}
+				.edgesIgnoringSafeArea(.all)
+			})
+			.fullScreenCover(isPresented: $isImagePickerPresented) {
+				ImagePicker(doneAction: { photo in
+					selectedImages.removeAll()
+					selectedImages = photo.filter { $0.url != nil }
+					updateAvata()
+				})
+			}
+			.onChange(of: countryCode, perform: { _ in
+				checkPhoneValid(text: phoneNumber)
+			})
+			.hideKeyboardOnTapped()
+			.toast(message: "Menu.Copy.Title".localized, isShowing: $isShowToastCopy, duration: Toast.short)
+			.onReceive(inspection.notice) { inspection.visit(self, $0) }
+			.alert("GroupChat.Warning".localized, isPresented: $showAlertPopup) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Profile/Views/UserProfileContentView.swift#L283"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>UserProfileContentView.swift:283</code></td>
    </tr>
  </tbody>
</table>